x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -362,7 +362,11 @@ func (k Keeper) HandleUndelegate(ctx sdk.Context, msg sdk.Msg, completionTime ti
 		return sdkerrors.Wrapf(sdkerrors.ErrLogic, "Undelegate message was not for a delegation account")
 	}
 
+	
 	undelegateAmt := undelegateMsg.Amount.Amount.Int64()
+	if undelegateAmt < 0 {
```

Comment:
```suggestion
	if undelegateAmt <= 0 {
```

Reply:
We probably want to fail it at 0, right? Or should/do we stop that case earlier so that the 0-balance redemption is never issued in the first place?